src/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -3399,6 +3399,9 @@
   <data name="InfoNoSerializableTypes" xml:space="preserve">
     <value>Assembly '{0}' does not contain any types that can be serialized using XmlSerializer.</value>
   </data>
+  <data name="InfoIgnoreType" xml:space="preserve">
+    <value>Ignoring '{0}'.</value>
```

Comment:
Is this supposed to be just one type of multiple types? Can you explain what this parameter is for?

Reply:
This is used in the warning message when we need bypass an unsupported type like generic type.